dependency-check-suppressions.xml

```diff
@@ -151,4 +151,11 @@
         <gav regex="true">.*</gav>
         <cve>CVE-2019-3795</cve>
     </suppress>
+    <suppress>
+        <notes><![CDATA[
+			Not applicable - A Remote Code Execution Vulnerability in Apache Tomcat
```

Comment:
can you please remove this. Silvia has made a fix already .https://github.com/hmcts/cmc-claim-store/pull/890